### PR TITLE
Test StarterSpell and avatar screenshots

### PR DESCRIPTION
**Pull request by ultrabot2022**
Fixes: https://github.com/Oneirocom/Magick/issues/715

Here are some screenshots for testing steps below:
step1]
First, click the "Create New" button on the home page.
step1

step2]
Second, select the Starter template and write down the spell name.
step2

step3]
Third, On the spell window, input any text in Playtest section. 
If you can see the echo back, that means that spell works correclty.
step3

step4]
On the main menu, please click avatar button to redirect. On the avatar page, you can see Character, Personality window, Chatbox window and Spell list.
Please select the Spell what you have just created.
step4

step5]
Next, please input any text in chatbox and click the send button. You can see Eliza's text and listen her voice with her Lip sync.
step5

step6]
Finally, If you want to change the Character, you can drag/drop any VRM file on the page
step6

These are summary for testing and current state.
Please DM with your testing and idea
Thanks for your time.

### CODE REVIEW

File: plugins/avatar/client/src/components/Chat.jsx [hunk 1]
```diff
@@ -176,9 +176,21 @@ export default function ChatBox({
           const arrayBuffer = await blob.arrayBuffer()
 
           self.startFromAudioFile(arrayBuffer)
-          setMessages((messages) => [...messages, agent + ": " + "hello"])
+          setMessages((messages) => [...messages, agent + ": " + value])
         })
 
+
+        // fetch('http://localhost:3030/api',
```
Comment (fire) [inline plugins/avatar/client/src/components/Chat.jsx:183]: Is this code needed? Why is it commented?

Comment (ultrabot2022) [reply plugins/avatar/client/src/components/Chat.jsx:183]: It is not needed for now.
Thank you

File: plugins/avatar/client/src/components/Chat.jsx [hunk 2]
```diff
@@ -167,7 +167,7 @@ export default function ChatBox({
         //   voices[voice]
 
         // fetch the audio file from ttsEndpoint
-        const ttsEndpoint = "https://ai-voice.webaverse.ai/tts?s=hello"
+        const ttsEndpoint = "https://ai-voice.webaverse.ai/tts?s="+value
```
Comment (fire) [inline plugins/avatar/client/src/components/Chat.jsx:170]: Switch to eleven labs.

File: plugins/avatar/client/src/components/Chat.jsx [hunk 3]
```diff
@@ -176,9 +176,21 @@ export default function ChatBox({
...
+        // {  
+        //   method: 'POST',  
+        //   headers: { 'Content-Type': 'application/json',},  
+        //   body: JSON.stringify({ 
+        //     id: '727574f9-88fd-47f6-be39-a52493da8514',    
+        //     apiKey: 'b2b14ea42a439cf138c55b450b9eddb5',    
+        //     content: 'Hello World'  
+        //   })
+        // })
+
```
Comment (fire) [inline plugins/avatar/client/src/components/Chat.jsx:193]:
```suggestion
```

**Review by fire — COMMENTED** [review]
Added some comments.

- [x] @michaelsharpe decided on keeping the webaverse api or not. "https://ai-voice.webaverse.ai/tts?s="
- [x] Remove comments.

**Comment by fire** [comment]
Pushed the changes required. Will do a test and will ask for approval. Looks good.

**Review by fire — APPROVED** [review]
@coffeeorgreentea

**Comment by fire** [comment]
Todo:

- [ ] Collect the screenshots for a tutorial
- [ ] Use elevenlabs text to speech

**Comment by ultrabot2022** [comment]
> Thanks for your kind mention.
I have some questions:
1. Collect the screenshots for a tutorial
    I need add more screenshots for tutorial?
2. Use elevenlabs text to speech
    I will use Elevenlabs API. 
    https://docs.elevenlabs.io/api-reference/text-to-speech
    Can you please provide me the API key?

**Comment by coffeeorgreentea** [comment]
I'm going to be pushing an eleven labs plug-in tonight we can use

**Comment by ultrabot2022** [comment]
Ok 👍

**Pull request merged**